test/integration/iso_test.go

```diff
@@ -89,3 +90,27 @@ func testPackages(t *testing.T) {
 	}
 
 }
+
+func testPersistence(t *testing.T) {
+	minikubeRunner := util.MinikubeRunner{
+		Args:       *args,
+		BinaryPath: *binaryPath,
+		T:          t}
+
+	for _, dir := range []string{
+		"/data",
+		"/var/lib/docker",
+		"/var/lib/cni",
+		"/var/lib/kubelet",
+		"/var/lib/localkube",
+		"/var/lib/rkt",
```

Comment:
we also mount /var/lib/boot2docker
we probably dont need to anymore, but maybe add it to the list or delete it from automount?